### ISSUE TITLE
Addon API broke with the 1.20.2 update

### ISSUE THREAD

**Issue by Paddyk45** [MeteorDevelopment/meteor-client#4124]
### Describe the bug

Using the (modified) mod template just throws some errors, then crashes

### Steps to reproduce

```bash
1. Download the addon template
2. Modify the gradle.properties:
2.1 Change minecraft_version to 1.20.2
2.2 Change the yarn mappings version to 1.20.2+build.4
2.3 Set meteor_version to 0.5.5-SNAPSHOT
3. Add the following to build.gradle under the dependencies closure:
modImplementation "baritone:fabric:1.20.2-SNAPSHOT"
4. Launch using runClient
```

### Link to crash report/log if applicable (https://mclo.gs)

https://mclo.gs/m6nBNVd

### Meteor Version

0.5.5

### Minecraft Version

1.20.2

### Operating System

macOS

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by RacoonDog** [comment]
the 0.5.5-SNAPSHOT pom is missing dependency information, so transitive dependencies are broken

**Comment by MineGame159** [comment]
Fixed in 3872a8eff3621549cde377a4989429cfdca317b2.
